lib/ansible/modules/clustering/kubevirt/kubevirt_raw.py

```diff
@@ -211,9 +211,7 @@ def execute_module(self):
 
         self.exit_json(**{
             'changed': changed,
-            'result': {
-                'results': results
-            }
+            'results': results
```

Comment:
Please update RETURN doc

Reply:
Just document it's `resutls` and it's list of the complex types, which are already documented.

Reply:
I think k8s has it same